src/Microsoft.TemplateEngine.IDE/Bootstrapper.cs

```diff
@@ -59,12 +59,29 @@ public void Register(Assembly assembly)
             EnvironmentSettings.SettingsLoader.Components.RegisterMany(assembly.GetTypes());
         }
 
+        [Obsolete("Use GetTemplates instead")]
         public async Task<IReadOnlyCollection<IFilteredTemplateInfo>> ListTemplates(bool exactMatchesOnly, params Func<ITemplateInfo, MatchInfo?>[] filters)
         {
             EnsureInitialized();
             return TemplateListFilter.FilterTemplates(await EnvironmentSettings.SettingsLoader.GetTemplatesAsync(default).ConfigureAwait(false), exactMatchesOnly, filters);
         }
 
+        /// <summary>
+        /// Gets list of available templates, if <paramref name="filters"/> are provided returns only matching templates.
+        /// </summary>
+        /// <param name="exactMatchesOnly">
+        /// true: templates should match all filters;
+        /// false:  templates should match any filter.
```

Comment:
Number of spaces after semicolon do not match.
Punctuation at the end of the lines do not match.